app/models/schools/attendance.rb

```diff
@@ -29,10 +29,11 @@ def save
     end
 
     def update_gitis
-      bookings_params.slice(*updated_bookings).each do |booking_id, _attended|
+      bookings_params.slice(*updated_bookings).each do |booking_id, attended|
         fetch(booking_id).tap do |booking|
-          Bookings::Gitis::EventLogger.write_later \
-            booking.contact_uuid, :attendance, booking
+          status = ActiveModel::Type::Boolean.new.cast(attended) ? :completed : :withdrawn
+          Bookings::Gitis::SchoolExperience.from_booking(booking, status)
```

Comment:
The old `EventLogger` deferred the writing to a delayed job (I _think_) - this will write it inline as part of the request, are we ok with that? if it's initiated by a user we probably want to shift it to a job